### PR TITLE
Handle Zero `CollateralPerOrder`

### PR DESCRIPTION
**Pull request by AurevoirXavier**
But I'm curious if we should add a `MaxCapacity`.

### CODE REVIEW

File: modules/fee-market/src/lib.rs [hunk 1]
```diff
@@ -503,7 +503,8 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 	}
 
 	fn collateral_to_order_capacity(collateral: BalanceOf<T, I>) -> u32 {
-		(collateral / T::CollateralPerOrder::get()).saturated_into::<u32>()
+		// If the `CollateralPerOrder` is zero, the maximum order capacity is `collateral`.
```
Comment (AsceticBear) [inline modules/fee-market/src/lib.rs:506]: If the `CollateralPerOrder` is zero, we should force return 0. which means all relayers have no available order capacity. the fee market is not available in this situation until we fix it in the runtime.

Comment (AurevoirXavier) [reply modules/fee-market/src/lib.rs:506]: If zero means not allow to relay, better to document that.

File: modules/fee-market/src/lib.rs [hunk 2]
```diff
@@ -69,6 +69,8 @@ pub mod pallet {
 		#[pallet::constant]
 		type MinimumRelayFee: Get<BalanceOf<Self, I>>;
 		/// The collateral relayer need to lock for each order.
+		///
+		/// Set this to zero to disable the relay.
```
Comment (AsceticBear) [inline modules/fee-market/src/lib.rs:73]: Suggest change to:
``` 
The collateral per order also often represents the maximum slash value for a single delayed order, please note that if this value is set to zero the fee market will be suspended.
```

**Review by AsceticBear — APPROVED** [review]
LGTM, please remember to draft a companion pull request to `darwinia-v0.12.4` branch after merging.

**Pull request merged**